### ISSUE TITLE
Missing file in NPM package?

### ISSUE THREAD

**Issue by bard** [bminixhofer/nnsplit#27]
I'm trying to import nnsplit in a JavaScript project, and webpack is failing with:

```
./node_modules/nnsplit/nnsplit.bundle/nnsplit_javascript_bg.wasm
Module not found: Can't resolve './nnsplit_javascript_bg.js' in '/tmp/experiment/node_modules/nnsplit/nnsplit.bundle'
```

Looking in `node_modules/nnsplit/nnsplit.bundle`, indeed the file `nnsplit_javascript_bg.js` is referenced by `package.json`, but missing from the filesystem.

(Not sure though whether that's the real culprit, as the nodejs example seems to work as intended.)

**Comment by bminixhofer** [comment]
This seems to be a valid issue. As you said I think `./nnsplit_javascript_bg.js` is missing. The Node.js build is built differently so it is not affected. Not sure *why* it is not included, I'll look into this.

**Comment by bminixhofer** [comment]
This was https://github.com/rustwasm/wasm-pack/issues/881 (fixed by https://github.com/rustwasm/wasm-pack/pull/839). I'm using a dev version of wasm-pack in CI now, that should fix the issue.

**Comment by bminixhofer** [comment]
`nnsplit_javascript_bg.js` is included as of v0.5.5. Thanks for reporting this!
